docs/guide-events.mdx

```diff
@@ -38,7 +38,7 @@ simply using `fireEvent.click` is worth it.
 We will describe a couple of simple adjustments to your tests that will increase
 your confidence in the interactive behavior of your components. For other
 interactions you may want to either consider using
-[`user-event`](ecosystem-user-event) or testing your components in a real
```

Comment:
I think that the only thing missing here is the `.mdx` suffix, can you please give it a try and add it instead of using the full site url? That won't help us on local env :)
Thanks!

Reply:
I've updated the code but I feel like it'll be better if I test this locally, just to make sure it all works.